src/app/modules/timesheet/components/invoice-form/invoice-form.component.html

```diff
@@ -211,35 +211,23 @@ <h4 class="invoice-form__half-block__title">Client</h4>
           </div>
         </div>
         <div class="invoice-form__half-block__input-wrapper__item form-group">
-          <label for="invoiceClientSIRENInput" class="invoice-form__half-block__input-wrapper__item__label">Numéro de
-            SIREN</label>
+          <label for="invoiceClientSIRENInput" class="invoice-form__half-block__input-wrapper__item__label">
+            Numéro de SIREN <span class="optional"> (optionnel)</span>
```

Comment:
You have the `optional` class, then a little `padding-left` in the css is easy and better than a whitespace :)